eng/dockerfile-templates/wcf/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# escape=`
+
+ARG REPO=mcr.microsoft.com/dotnet/framework/aspnet
+FROM $REPO:{{PRODUCT_VERSION}}-{{OS_VERSION}}
+
+# Install Windows components required for WCF service hosted on IIS
+{{if OS_VERSION_NUMBER = "ltsc2016" || OS_VERSION_NUMBER = "ltsc2019" || OS_VERSION_NUMBER = "1903" || OS_VERSION_NUMBER = "1909" || OS_VERSION_NUMBER = "2004"
+:RUN Add-WindowsFeature NET-WCF-TCP-Activation45; `
+    Add-WindowsFeature NET-WCF-HTTP-Activation45; `
+    Add-WindowsFeature Web-WebSockets
+^else
+:RUN dism /Online /Quiet /Enable-Feature /All /FeatureName:WCF-HTTP-Activation45 /FeatureName:WCF-TCP-Activation45 /FeatureName:IIS-WebSockets
+}}
+# Enable net.tcp protocol for default web site on IIS
+{{if OS_VERSION_NUMBER = "ltsc2016" || OS_VERSION_NUMBER = "ltsc2019" || OS_VERSION_NUMBER = "1903" || OS_VERSION_NUMBER = "1909" || OS_VERSION_NUMBER = "2004"
+:RUN windows\system32\inetsrv\appcmd.exe set app 'Default Web Site/' /enabledProtocols:"http,net.tcp"
```

Comment:
This seems like a non-functional difference compared to 20H2 that could simply be updated for these versions to provide consistency with 20H2.

Reply:
The trouble is that pre 20H2, the SHELL is set to PowerShell.  I am not finding a syntax that works with both shells that makes sense going forward.  I am open to suggestions incase I missed something obvious.

Reply:
Ok, I see now.